userguide/getting-started/quick_start_with_kaui.adoc

```diff
@@ -0,0 +1,189 @@
+
+For simplicity, we will create an offline payment method - checks - for the account we created in step 3.  
+
+. Search for and open the customer account. (If you still have the account open from the previous section, you can skip this step.) 
+.. On the main Kaui page, click in the center text box and press the Enter key (or click the search icon).
++
+image::ShowSearchHomepage.png[width=650,align="center"]
++
+Kaui displays the accounts you have created so far for the tenant. 
++
+.. Click on an account, which opens the Account page for the customer. 
+. On the lower right, click the *+* sign next to *Payment Methods*. 
++ 
+image::payment-method-empty.png[width=450,align="center"]
++
+Kaui opens the Add New Payment Method screen.
++
+. Verify that `__EXTERNAL_PAYMENT__` appears in the *Plugin name* field and check the *Default payment method* checkbox.  
++
+image::add-new-payment-method-cropped.png[width=450,align="center"]
++
+. Click the *Save* button.
++ 
+Kaui displays the new payment method as `__EXTERNAL_PAYMENT__` in the *Payment Method* section. The filled star to its left indicates it is the _default_ payment method: 
++
+image::payment-method-with-default-selected.png[width=350,align="center"] 
+
+[[step5]]
+== Step 5. Set Up a Subscription for the Account
+
+You are now ready to create a subscription for the customer. 
+
+. On the customer Account page, click *Subscriptions* on the sub-menu. 
++ 
+image::subscription-blank.png[width=850,align="center"]
++
+. Next to *Subscription Bundles*, select the plus icon ( + ).
++
+[NOTE]
+*Note:* A subscription bundle is a collection of subscriptions. A bundle contains at leaset one base subscription and zero or more add-ons.
++
+Kill Bill opens the Add New Subscription screen. 
++
+. In the *Plan* field, select *standard-monthly*. 
++
+image::add-new-subscription-labeled.png[width=450,align="center"]
++
+. Click *Save*.
+. The customer is now subscribed to the plan, and it is displayed on the Subscriptions page: 
++  
+image:subscription-created.png[width=850,align="center"]
+
+[[step6]]
+== Step 6. View the Generated Invoice
+
+To view the invoice that Kill Bill automatically generated for the subscription (step 5):  
+
+. On the account page, click *Invoices* on the sub-menu: 
++
+image::invoices.png[width=850,align="center"]
++
+[NOTE]
+*Note: Kill Bill immediately generated the invoice for the first month of the subscription because the plan (on which the subscription in based) has no trial period and because billing is performed _in advance_ by default.
++
+. Click on the invoice number to see the invoice details: 
++
+image::invoice-detail.png[width=650,align="center"]
++
+If a payment gateway had been configured with Kill Bill _and_ the account had a valid payment method with the gateway, Kill Bill would have immediately paid this invoice. In that case, you would see the relevant payment information under the invoice detail. 
```

Comment:
Do you think adding a screenshot here that displays the payment screen would help? Since you have explained the steps to add `__EXTERNAL_PAYMENT__` as the default payment method?

Reply:
I am running KB on my local machine. I set up the default `__EXTERNAL_PAYMENT__` but the invoice did not get automatically paid. 

Same scenario on demo.killbill.io, but Kill Bill DOES auto-pay the invoice.

Is there some default setting I'm not aware of when you download and install KB/Kaui locally? I worked on trying to figure this out for a while, but came up short.

Reply:
No, not that I'm aware of. If `__EXTERNAL_PAYMENT__` is set as the default payment method, the invoice should get automatically paid. Do you by any chance have the `AUTO_INVOICING_OFF` tag set on the account?

Reply:
I started with a new tenant, and now everything works fine, so I updated the screenshot.